ui/copy-fusor-ember-cli-to-assets

```diff
@@ -0,0 +1,20 @@
+ #!/bin/bash
+
+cp ../fusor-ember-cli/dist/assets/vendor.css          ./app/assets/stylesheets/fusor_ui/vendor-fusor.css;
+cp ../fusor-ember-cli/dist/assets/fusor-ember-cli.css ./app/assets/stylesheets/fusor_ui/;
+
+cp ../fusor-ember-cli/dist/assets/vendor.js       ./app/assets/javascripts/fusor_ui/vendor-fusor.js;
+cp ../fusor-ember-cli/dist/assets/fusor-ember-cli.js   ./app/assets/javascripts/fusor_ui/;
+
+cp ../fusor-ember-cli/dist/assets/rhci-rhev-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/rhci-openstack-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/rhci-cloudforms-640-210.png  ./app/assets/images/;
+
+cp ../fusor-ember-cli/dist/assets/ovirt-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/rdo-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/manageiq-640-210.png  ./app/assets/images/;
+
+cp ../fusor-ember-cli/dist/fonts/*.*  ./public/fonts/;
+
+git add .;
+git commit -m "moved assets from /fusor-ember-cli/dist to /ui/app/assets";
```

Comment:
output from `fusor-ember-cli/dist` is copied to `ui/app/assets`. The `ui` directory is the rails engine that is loaded into Satellite.
```ruby
gem 'fusor_ui', :git => 'https://github.com/fusor/ui.git'
```